maven-release-plugin/src/main/java/org/apache/maven/plugins/release/BranchReleaseMojo.java

```diff
@@ -199,12 +199,12 @@
     private String projectVersionPolicyId;
 
     /**
-     * The role-hint for the NamingPolicy implementation used to calculate the project branch and tag names.
+     * The role-hint for the NamingPolicy implementation used to calculate the project names.
      *
      * @since 3.0.0
      */
-    @Parameter( defaultValue = "default", property = "projectNamingPolicyId" )
-    private String projectNamingPolicyId;
+    @Parameter( defaultValue = "default-branch", property = "projectBranchNamingPolicyId" )
```

Comment:
defaultValue should stay "default", property should stay "projectNamingPolicyId", since you will never run `release:prepare` and `release:branch`. Only the name of the field has to be different, so you can configure both inside your pom.